packages/kit/src/runtime/client/fetcher.js

```diff
@@ -26,7 +26,12 @@ if (import.meta.env.DEV) {
 		const url = input instanceof Request ? input.url : input.toString();
 		const stack = /** @type {string} */ (new Error().stack);
 
-		const heuristic = can_inspect_stack_trace ? stack.includes('load_node') : loading;
+		// check if fetch was called via load_node. the lock method only checks if it was called at the
+		// same time, but not necessarily if it was called from `load`
+		// we use just the filename as the method name sometimes does not appear on the CI
+		const heuristic = can_inspect_stack_trace
+			? stack.includes('src/runtime/client/client.js')
+			: loading;
```

Comment:
is this an async thing? because `can_inspect_stack_trace` works by checking to see if a newly created stack trace includes the `check_stack_trace` function

Reply:
No idea. `load` is contained in the stack trace, but not the others:
```
@http://localhost:5173/@fs/Users/runner/work/kit/kit/packages/kit/src/runtime/client/fetcher.js:26:49
@http://localhost:5173/src/routes/load/window-fetch/incorrect/+page.js:3:25
load@http://localhost:5173/src/routes/load/window-fetch/incorrect/+page.js:2:28
@http://localhost:5173/@fs/Users/runner/work/kit/kit/packages/kit/src/runtime/client/client.js:653:41
```

I'm on mobile at the moment so can't easily check whether that's because of async or not, but I'm not sure it'd change the fix? It would be interesting to know though

Reply:
As far as I can tell, it's not related to async vs sync. `load` is async and showed up in the stack trace, but the lines before and after are async and the methods didn't show up there.